### ISSUE TITLE
Gamecube Emulation

### ISSUE THREAD

**Issue by moiez0** [emulatorjs/emulatorjs#187]
Hey.

https://github.com/ethanaobrien/emulatorjs/issues/151

What do you mean by "no retroarch gamecube implementation" ?

I don't quite understand. If you mean retroarch does not have a gamecube core then it does, it is just not available on the core downloader as their bot got hacked and some of the cores got removed. There are still archives available.

Please could you elaborate, thanks!

Also, I'm wondering would the if the dolphin-libretro could possibly work?
https://docs.libretro.com/library/dolphin/

**Comment by ethanaobrien** [comment]
This isnt possible, there is no emscripten build target for the dolphin core.

The requirements (I got from the link you sent) state


- OpenGL/Open GL ES 3.0 or higher for the OpenGL renderer.

There is no opengl in the browser, the browser uses webgl so this entire module would need to be translated

- Vulkan for the Vulkan renderer.

This is technically optional, but still not avaliable

- Direct3D 11 for the Direct3D 11 renderer.

And again, not possible


The technologies in the dolphin core are not built to run on the web. To put it simply, this isnt possible, sorry. Theres nothing we can do about this.

Even if we got it to run, the nintendo 64 core can barely run, the dolphin core (especially wii games) would be like 20x worse
